src/module/actors/npc-sheet.ts

```diff
@@ -274,7 +283,8 @@ export class NPCActorSheet extends ActorSheet<
       }
     }
   }
-}
 
+
+}
```

Comment:
Don't change formatting for no reason, scrap changes here.

Reply:
cleaned up. i will be more conscious of this going forward. there will likely be a few in some coming PRs as they are hard to determine when this happened.